packages/component/src/Activity2/CarouselFilmStrip.js

```diff
@@ -0,0 +1,145 @@
+import { css } from 'glamor';
+import { Context as FilmContext } from 'react-film';
+import classNames from 'classnames';
+import React from 'react';
+
+import Avatar from './Avatar';
+import Bubble from './Bubble';
+import Context from '../Context';
+import Timestamp from './Timestamp';
+
+import textFormatToContentType from '../Utils/textFormatToContentType';
+
+const ROOT_CSS = css({
```

Comment:
would be nice if this was in css/scss and imported instead of being inline in the ts/js.

Reply:
This is actually [glamor](https://github.com/threepointone/glamor), a JavaScript-based CSS.

I personally don't like import CSS/SCSS, because it will force the package to be Webpack-ed on build. With multiple layers of Webpack, it will cause the build to slowdown significantly.

However, I could separate the CSS into a separate JS file. All `ROOT_CSS` you see in the JS files are minimal non-customizable CSS, mostly DOM-related layout rules, e.g. `display: 'flex'`. Instead of other customizable CSS like color or padding.

So our approach is more towards microservices build architecture:
- Use Babel for transpiling everything (including TypeScript)
- Use TypeScript only for typechecking
- Use Webpack only for bundling (i.e. no `style-loader`, no other loaders)